backend/src/services/UserService.ts

```diff
@@ -1,53 +1,129 @@
-
+/**
+ * -----------------------------------------------------------------------------
+ * Imports
+ * -----------------------------------------------------------------------------
+ */
+import { resolve } from 'path';
```

Comment:
unused?

Reply:
well, auto-added by the IDE so I don't know